guides/release/components-update/html-is-html.md

```diff
@@ -0,0 +1,214 @@
+  University of Awesome<br />
+  Bobtown, CA 99999,<br />
+  USA<br />
+  <strong>Tel</strong>: 123-456-7890<br />
+  <strong>Email</strong>: no_reply@example.com
+</address>
+
+<p class="sender-column"><time datetime="2016-01-20">20 January 2016</time></p>
+
+<address>
+  <strong>Miss Eileen Dover</strong><br />
+  4321 Cliff Top Edge<br />
+  Dover, CT9 XXX<br />
+  UK
+</address>
+
+<h1>Re: Eileen Dover university application</h1>
+
+<p>Dear Eileen,</p>
+
+<p>
+  Thank you for your recent application to join us at the University of
+  Awesome's science faculty to study as part of your
+  <abbr title="Doctor of Philosophy">PhD</abbr> next year. I will answer your
+  questions one by one, in the following sections.
+</p>
+
+<h2>Starting dates</h2>
+
+<p>
+  We are happy to accomodate you starting your study with us at any time,
+  however it would suit us better if you could start at the beginning of a
+  semester; the start dates for each one are as follows:
+</p>
+
+<ul>
+  <li>First semester: <time datetime="2016-09-09">9 September 2016</time></li>
+  <li>Second semester: <time datetime="2017-01-15">15 January 2017</time></li>
+  <li>Third semester: <time datetime="2017-05-02">2 May 2017</time></li>
+</ul>
+
+<p>
+  Please let me know if this is ok, and if so which start date you would prefer.
+</p>
+
+<p>
+  You can find more information about
+  <a
+    href="http://www.example.com"
+    title="table of awesome university important dates"
+    >important university dates</a
+  >
+  on our website.
+</p>
+
+<h2>Subjects of study</h2>
+
+<p>
+  At the Awesome Science Faculty, we have a pretty open-minded research facility
+  — as long as the subjects fall somewhere in the realm of science and
+  technology. You seem like an intelligent, dedicated researcher, and just the
+  kind of person we'd like to have on our team. Saying that, of the ideas you
+  submitted we were most intrigued by are as follows, in order of priority:
+</p>
+
+<ol>
+  <li>
+    Turning H<sub>2</sub>O into wine, and the health benefits of Resveratrol
+    (C<sub>14</sub>H<sub>12</sub>O<sub>3</sub>.)
+  </li>
+  <li>
+    Measuring the effect on performance of funk bassplayers at temperatures
+    exceeding 30°C (86°F), when the audience size exponentially increases
+    (effect of 3 × 10<sup>3</sup> increasing to 3 × 10<sup>4</sup>.)
+  </li>
+  <li>
+    <abbr title="HyperText Markup Language">HTML</abbr> and
+    <abbr title="Cascading Style Sheets">CSS</abbr> constructs for representing
+    musical scores.
+  </li>
+</ol>
+
+<p>
+  So please can you provide more information on each of these subjects,
+  including how long you'd expect the research to take, required staff and other
+  resources, and anything else you think we'd need to know? Thanks.
+</p>
+
+<h2>Exotic dance moves</h2>
+
+<p>
+  Yes, you are right! As part of my post-doctorate work, I <em>did</em> study
+  exotic tribal dances. To answer your question, my favourite dances are as
```

Comment:
I am not sure that referencing exotic tribal dances is appropriate for Ember content. Also, we use American English spelling for website content, please run a spell check.

Reply:
You can use `npm run lint:md` to run the spellcheck locally when you are ready. Anything that needs to be added to the dictionary can be added in `local.dic`

Reply:
@MelSumner @jenweber pointed this out to me, and I'm embarrassed to say that I didn't really read the content. I copied it directly from MDN with the mistaken assumption that its presence on MDN would make it unobjectionable.